src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -100,9 +101,30 @@ private static OnnxTransform Create(IHostEnvironment env, ModelLoadContext ctx)
             if (!ctx.TryLoadBinaryStream("OnnxModel", r => modelBytes = r.ReadByteArray()))
                 throw env.ExceptDecode();
 
-            var inputColumn = ctx.LoadNonEmptyString();
-            var outputColumn = ctx.LoadNonEmptyString();
-            var args = new Arguments() { InputColumn = inputColumn, OutputColumn = outputColumn };
+            bool isMultiOutput = ctx.Header.ModelVerReadable > 0x00010001;
```

Comment:
>ModelVerReadable [](start = 44, length = 16)

ModelVerWritten

Reply:
Good catch. That sounds more correct -- updated. TF probably needs updating as well.

---
In reply to: [232842442](https://github.com/dotnet/machinelearning/pull/1586#discussion_r232842442) [](ancestors = 232842442)